xml/depl_ha_recovery.xml

```diff
@@ -130,47 +138,40 @@
    </step>
    <step>
     <para>Re-register the node with &crow; by executing:</para>
-    <screen>&prompt.root;service crowbar_join start</screen>
+    <screen>&prompt.root;<command>systemctl</command> start crowbar_join</screen>
    </step>
-   <!--
-   taroth 2016-02-24: aspiers, is the following still (or also?) needed?
-   copied from depl_nodex.xml, l.625-->
-   <!--
-    <listitem>
-    <para>
-     On the &admserv;, run the following command to update the status
-     of the node specified with <replaceable>NODE</replaceable>.
-    </para>
-    <screen>crowbar crowbar transition <replaceable>NODE</replaceable> ready</screen>
-   </listitem>
-   -->
    <step>
     <para>Log in to one of the <emphasis>other</emphasis>
```

Comment:
IIRC, this step was to ensure `haproxy.cfg` is correct (Crowbar might have dropped nodes from it if they were down for long enough), in which case it needs to be done for *all* other nodes in the cluster.  Right @vuntz?